### ISSUE TITLE
Provide dockerfile name

### ISSUE THREAD

**Issue by danger89** [AkhileshNS/heroku-deploy#75]
Besides the default 'Dockerfile' filename, you can rename your Dockerfile to other names, which should be possible.

However, I do not see any way to provide another Dockerfile name (eg. `myspecialDockerfile`) as input. With the `docker` command you can use `docker build -f myspecialDockerfile .` (see `-f` flag).

Regards,
Melroy

**Comment by ColeRutledge** [comment]
> Besides the default 'Dockerfile' filename, you can rename your Dockerfile to other names, which should be possible.
> 
> However, I do not see any way to provide another Dockerfile name (eg. `myspecialDockerfile`) as input. With the `docker` command you can use `docker build -f myspecialDockerfile .` (see `-f` flag).

Agreed. It's a common convention to use Dockerfile.prod or Dockerfile.dev so it would be nice to have some control over that. Along the same lines, it would be great to provide options for setting the build context and the path to the Dockerfile, so we would have control over the naming convention and location of our Dockerfile vs it always being required to be in the root.

Thanks for the work you've done on this action! Very helpful! 👍
